applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.h

```diff
@@ -39,9 +42,29 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) StressStrainUtilities
                                                 bool                       UseHenckyStrain,
                                                 std::size_t                VoigtSize);
 
+    static constexpr std::size_t GetVoigtSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? GetVoigtSize3D() : GetVoigtSize2D();
+    }
+
+    static constexpr std::size_t GetStressTensorSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? STRESS_TENSOR_SIZE_3D : STRESS_TENSOR_SIZE_2D;
+    }
+
+    static const Vector& GetVoigtVector2D();
+    static const Vector& GetVoigtVector3D();
+
+    static constexpr SizeType GetVoigtSize2D() { return VOIGT_SIZE_2D_PLANE_STRAIN; }
+
+    static constexpr SizeType GetVoigtSize3D() { return VOIGT_SIZE_3D; }
```

Comment:
As you mentioned during standup, this is necessary for some constexpr variables. In my opinion it's not worth it to have duplicate logic (since the stress state policy is the one responsible for returning the right size/vector). Maybe after @avdg81 has taken a look we can have a few minute call to align probably quicker than via comments?)